argo-messaging.spec

```diff
@@ -0,0 +1,63 @@
+#debuginfo not supported with Go
+%global debug_package %{nil}
+
+Name: argo-messaging
+Summary: ARGO Messaging API for broker network
+Version: 0.9.0
+Release: 1%{?dist}
+License: ASL 2.0
+Buildroot: %{_tmppath}/%{name}-buildroot
+Group:     EGI/SA4
```

Comment:
I think this is a remnant we have in the argo-web-api as well. I would suggest better to use

```
Group: Applications/Publishing
```

What do others think?